lifecycle_test.go

```diff
@@ -919,7 +919,7 @@ func TestPrintContainerLogsOnError(t *testing.T) {
 				break
 			}
 
-			t.Fatal("Read Error:", err)
+			require.NoErrorf(t, err, "Read Error")
```

Comment:
bug: the err was already not nil in https://github.com/testcontainers/testcontainers-go/pull/2857/files#diff-8c1de2f5177b67b15485d779c1510b2fa4abeb3cd1e82c18d43d0c2e7871ed19R917